frontend/src/utils/items.ts

```diff
@@ -81,6 +81,16 @@ export function isValidMD5(input: string): boolean {
   return /[\dA-Fa-f]{32}/.test(input);
 }
 
+/**
+ * Checks if an item is a local item.
+ *
+ * @param item - The item to be checked.
+ * @returns - A boolean representing whether the item is a local item
+ */
+export function isLocalItem(item: BaseItemDto): boolean {
+  return item?.Id?.indexOf('local') === 0;
+}
```

Comment:
Why is this needed?

Reply:
Legacy code? It also used in jf-web so I'm basically just adding it here

Reply:
@noaione I think we can safely remove this, if it works without it. There's no point in adding obscure code (server and web are full of them since the fork) if it works without. If later on there's a related bug with it, we can always fix it.

I recall hearing about that property before, and I think it was related for offline usage (so cases when the client had downloaded content for offline playback). That never ended up getting implemented.

We can construct some stuff thinking forward and taking into account offline support, but assuming we can't do instant mixes because the items are cached offline *it's too much*